recipe/meta.yaml

```diff
@@ -7,49 +7,58 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/confluentinc/confluent-kafka-python/archive/refs/tags/v{{ version }}.tar.gz
   sha256: ee099702bd5fccd3ce4916658fed4c7ef28cb22e111defb843d27633100ff065
-  patches:
+  patches:  # [win]
     - 0001-Fix-setup-for-windows.patch  # [win]
 
 build:
   number: 0
+  skip: True # [py<38 or s390x]
   script:
     - export CFLAGS="${CFLAGS} -I${PREFIX}/include"  # [unix]
     - export LDFLAGS="${LDFLAGS} -I${PREFIX}/lib"    # [unix]
     - set CFLAGS="%CFLAGS% /I%LIBRARY_INC%"          # [win]
     - set LDFLAGS="%LDFLAGS% /I%LIBRARY_LIB%"        # [win]
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - m2-patch    # [win]
   host:
     - python
     - setuptools
     - pip
-    - librdkafka >={{ version }}
+    - wheel
+    - librdkafka {{ version }}
+
   run:
     - python
-    - librdkafka >={{ version }}
+    - librdkafka {{ version }}
```

Comment:
Why is an exact pinning necessary here? From what I understand, you can use a later version during runtime.

Reply:
because it's required [here](https://github.com/confluentinc/confluent-kafka-python/blob/6dd1e5784bcb77297015ca1e4636ef6c6ec2af34/CHANGELOG.md?plain=1#L14)

Reply:
The changelog doesn't say that it must be exactly `2.2.0`. There are code sections that suggest that later versions are also acceptable.

https://github.com/confluentinc/confluent-kafka-python/blob/6dd1e5784bcb77297015ca1e4636ef6c6ec2af34/src/confluent_kafka/src/confluent_kafka.h#L62

https://github.com/confluentinc/confluent-kafka-python/blob/6dd1e5784bcb77297015ca1e4636ef6c6ec2af34/src/confluent_kafka/src/confluent_kafka.c#L2020-L2026

Reply:
do you suggest we add
`- librdkafka >={{ version }}`
back into it?

Reply:
I see @cbouss did it. I will resolve this.

Reply:
Based off updating this package, it doesn't even seem that `python-confluent-kafka` can be built properly without having the equivalent recent version of `librdkafka`. This is why I set the version equivalent to  `python-confluent-kafka`.

Reply:
Yes, however `- librdkafka >={{ version }}` is a run dependency, not a host dependency. In the host dependency we have exact pinning on the current version, meaning building won't be an issue. The run dependency simply reflects what the upstream project seems to imply (see @marcoesters comments).